Makefile

```diff
@@ -240,13 +240,17 @@ update-libcalico:
 git-status:
 	git status --porcelain
 
+git-config:
+	git config --global user.name "Semaphore Automatic Update"
+	git config --global user.email marvin@tigera.io
```

Comment:
Could this clobber a developer's personal Git config?

Reply:
This rule is only meant to be executed by the commit-pin-updates target, which is invoked by CI when run on the master branch. If the dev decides to run this manually, it will definitely overwrite his/her config.